### PR TITLE
Added borderless icon class

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

File: src/app/public/styles/themes/modern/_buttons.scss [hunk 1]
```diff
@@ -77,6 +77,16 @@
     );
   }
 
+  .sky-btn-icon-borderless {
+    @include sky-theme-modern-button-variant(
+      $sky-theme-modern-font-deemphasized-color,
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/styles/themes/modern/_buttons.scss:82]: Is this color correct? It's barely perceptible in the visual tests.

Comment (Blackbaud-ToddRoberts) [reply src/app/public/styles/themes/modern/_buttons.scss:82]: the default theme version is super light, that's why we darkened it for modern theme. it should be the normal deemphasized color, which looks like it is to me

Comment (Blackbaud-ToddRoberts) [reply src/app/public/styles/themes/modern/_buttons.scss:82]: tho maybe i am looking at the wrong button?

**Comment by Blackbaud-AlexKingman** [comment]
Hold: Waiting on specs for borderless icon class.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-SteveBrush @Blackbaud-ToddRoberts 
Updated for new design spec: Borderless buttons should be 26x26. Also added support for default mode.

This is ready for review again.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-borderless-icon)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Pull request merged**